subCommand.go

```diff
@@ -212,6 +216,13 @@ func (sc *Subcommand) parse(p *Parser, args []string, depth int) error {
 
 	// if a command is parsed, its used
 	sc.Used = true
+	debugPrint("used subcommand", sc.Name, sc.ShortName)
```

Comment:
i could probably make this smarter so that we don't add positionals for the short or long name that wasn't used.  Right now, we add a positional for both the short and long names anytime the subcommand is invoked.